source/wp-content/themes/wporg-showcase-2022/src/site-meta-list/style.scss

```diff
@@ -1,39 +1,31 @@
-.wp-block-wporg-site-meta-list dl {
-	margin: 0;
-}
-
-.wp-block-wporg-site-meta-list dl > div {
-	display: grid;
-	grid-template-columns: 1fr 1fr;
-}
+.wp-block-wporg-site-meta-list {
```

Comment:
It gets a bit tight at `895px`.

<img width="267" alt="Screenshot 2023-08-09 at 9 51 31 AM" src="https://github.com/WordPress/wporg-showcase-2022/assets/1657336/f0819995-2f04-4a2b-97b8-fce291ac45c7">